src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -343,14 +343,6 @@ public void canGetMultipleRequests()
       .withTags(new RequestBuilder.Tags(asList(NEW_TAG, IMPORTANT_TAG)))
     );
 
-    requestsClient.create(new RequestBuilder()
```

Comment:
I think it would make this example clearer if each request being created defined the type. That would help explain why there is only a certain number, e.g. one of each type

Reply:
I am not sure I follow you, what should I do exactly?

Reply:
Apologies, I only meant adding a `.recall()`, `.hold()` etc to each of the requests being built with the builder

Reply:
Done